ddtrace/sampler.py

```diff
@@ -175,8 +176,18 @@ def __init__(
             rate_limit = int(get_env("trace", "rate_limit", default=self.DEFAULT_RATE_LIMIT))  # type: ignore[arg-type]
 
         # Ensure rules is a list
-        if not rules:
-            rules = []
+        if rules is None:
+            env_sampling_rules = get_env("trace", "sampling_rules")
+            if env_sampling_rules:
+                try:
+                    rules = self._parse_rules_from_env_variable(env_sampling_rules)
+                except ValueError as e:
+                    raise ValueError(
+                        "Unable to set DD_TRACE_SAMPLING_RULES={}".format(env_sampling_rules),
```

Comment:
```suggestion
                        "Unable to parse DD_TRACE_SAMPLING_RULES={}".format(env_sampling_rules),
```

Reply:
My only issue on this is that when it catches the ValueError exception that isn't a JSON config such as if a sampling rate isn't between 0 & 1, this really isn't a parsing issue. Though though printing the underling exception negates that a little.
```
ValueError: ('Unable to **parse** DD_TRACE_SAMPLING_RULES=[{"sample_rate":2.0,"service":"abc","name":"abc"}]', ValueError('SamplingRule() must be greater than or equal to 0.0 and less than or equal to 1.0', {'sample_rate': 2.0, 'service': 'abc', 'name': 'abc'}))
```

This is a pretty small nit pick so I'm happy to have it be parse if you think that's a better approach.

Reply:
This is a good point. Not sure I feel _that_ strongly about it.

"set" does make sense, could also do "read" maybe ?

again, no strong opinion.

Reply:
Similar not very strong opinion it was just something I noticed when testing. I'm going to change it to the following
`Unable to read DD_TRACE_SAMPLING_RULES={}`.